docs/develop/dotnet/schedules.mdx

```diff
@@ -0,0 +1,58 @@
+---
+id: schedules
+title: Schedules
+sidebar_label: Schedules
+description: Schedules
+keywords:
+  - sdk
+  - dotnet
+  - schedules
+tags:
+  - sdk
+  - dotnet
+  - schedules
+---
+
```

Comment:
We are inconsistent with whether the top of the page has a summary of what's on the page